python/cuml/benchmark/run_benchmarks.py

```diff
@@ -142,13 +146,35 @@ def extract_param_overrides(params_to_sweep):
         action='store_true',
         help='Throw exception on a failed benchmark',
     )
+    parser.add_argument(
+        '--algorithms',
```

Comment:
can we call it `print-algorithms` or something similar? This looks like you're specifying which algorithms to run.